Add notes for non-gsutil users. (#1388)
spinnaker/spinnaker.github.io: guides/operator/custom-boms.md

subcomponent `${SUBCOMPONENT}` at version `${SUBCOMPONENT_VERSION}` can be
found at `gs://${BUCKET}/${SUBCOMPONENT}/${SUBCOMPONENT_VERSION}/${PROFILE}`.

> If Halyard is unable to read from the GCS bucket, please see the troubleshooting

Reviewer: ```suggestion
If Halyard can't read from the GCS bucket, please see the troubleshooting
```

> instructions [here](https://www.spinnaker.io/setup/quickstart/faq/#halyard-times-out-during-a-config-change).

Reviewer: Instead of "...troubleshooting instructions [here]().", maybe just "...[troubleshooting instructions]()."
